packages/api-format/src/codec/U16.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2017-2018 @polkadot/api-format authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import BN from 'bn.js';
+
+import BaseNumber from './base/Number';
+
+export default class U16 extends BaseNumber {
```

Comment:
Only have u16 implemented here since it is all we need for this specific one. (u8, u32, u64, u128 is basically exactly the same as this one, just with the bitlength updated)